GUIDE.md

```diff
@@ -0,0 +1,105 @@
+# Editing mappings
+
+## Starting up
+
+After running a mappings task (either `mappings` or `mappingsUnpicked` via Gradle), you'll be greeted with this imposing Swing GUI:
+![an image of enigma's main window, devoid of life. the "deobfuscated classes", "obfuscated classes", and "structure" dockers are open on the sides, and nothing is displayed in the centre of the screen.](https://github.com/ix0rai/quilt-mappings/assets/66223394/fc569c2c-7b6e-4d22-9199-dd9b52afd8ca)
```

Comment:
```suggestion
![An image of enigma's main window, devoid of life. The "deobfuscated classes", "obfuscated classes", and "structure" dockers are open on the sides, and nothing is displayed in the centre of the screen.](https://github.com/ix0rai/quilt-mappings/assets/66223394/fc569c2c-7b6e-4d22-9199-dd9b52afd8ca)
```
Or is capitalization unconventional in image descriptions for some reason?

Reply:
nope, I was just lazy on my capitalisation pass apparently